### PR TITLE
[DO NOT MERGE] Byzantine refactor

### PR DESCRIPTION
**Pull request by JohnsonArnek**

### CODE REVIEW

**Comment by ColdSlav** [comment]
Mind giving a short explanation of what this PR is about, and what it adds/fixes?

**Comment by JohnsonArnek** [comment]
This is the Byzantium Mini-Rework? I tested it enough to see that it works, and it shouldn't break anything.

**Review by ColdSlav — COMMENTED** [review]
Why is there a second "GRE2" file? why not just put it on the GRE file?

**Comment by JohnsonArnek** [comment]
Honestly it was bugging out and after I seperated it. It just worked do check (the localization though it seesm to not be perfect in the migration?)

File: TGC/decisions/GRE2.txt [hunk 1]
```diff
@@ -0,0 +1,693 @@
+political_decisions = {
+
+	#Description: Byzantine ambitions towards the Levant.
+		reclaim_levant = {
+		potential = {
+			 primary_culture = greek
```
Comment (joelAVS) [inline TGC/decisions/GRE2.txt]: There's no point in checking for tag and primary culture, the BYZ's primary culture is always greek, so the check is redundant, unless there's some primary culture change shenanigans. This same issue happens in pretty much all events.

The fewer checks made, the less performance it costs, it's small but we gotta squeeze the most at any given time

**Comment by JohnsonArnek** [comment]
Let me fix the localization really quickly

File: TGC/decisions/GRE2.txt [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+		 ai_will_do ={ factor = 1 }
+	 }
+	 #Description: Byzantine ambitions towards Messopotamia.
+		reclaim_messopotamia = {
+		potential = {
+			 primary_culture = greek
+			 tag = BYZ
+			 NOT = { has_country_flag = retreated_from_anatolia }
+			  has_country_flag = legacy_of_rome
+			  has_country_flag = no_turks_in_anatolia
```
Comment (joelAVS) [inline TGC/decisions/GRE2.txt]: The country_flag byzantine_levant needs the country_flag legacy_of_rome to exist, there's no point in checking for legacy_of_rome here again, as it was already checked at the start of the event chain, unless you can somehow lose this flag as a possible outcome of the event. Most event chains can be controlled by one flag once it gets rolling.

There's other instances of this in the code and with other flags, once again the fewer checks we make the better it is for performance

**Comment by JohnsonArnek** [comment]
[00_decisions.csv](https://github.com/rderekp/The-Grand-Combo/files/10796868/00_decisions.csv)

**Comment by JohnsonArnek** [comment]
This is fair, honestly I mostly modified the existing code originally.

**Comment by JohnsonArnek** [comment]
[00_decisions(1).csv](https://github.com/rderekp/The-Grand-Combo/files/10797037/00_decisions.1.csv)
And I am legit incompetent when it comes to github
This should be the csv without anything changed from the original and with my addition in the localization at the end

File: TGC/decisions/GRE2.txt (hunk 2 above)
Comment (JohnsonArnek) [reply TGC/decisions/GRE2.txt]: With my latest comit it should now be fixed

Comment (joelAVS) [reply TGC/decisions/GRE2.txt]: My nit picks are fixed indeed.

**Comment by JohnsonArnek** [comment]
Some Extra Content Added.

**Comment by JohnsonArnek** [comment]
This is finished

**Comment by emprahtheodosius** [comment]
this was manually reviewed, revamped and rebalanced

**Pull request closed without merging**